recipe/meta.yaml

```
@@ -1,27 +1,25 @@
 {%set name = "Routes" %}
 {% set version = "2.5.1" %}
-{%set compress_type = "tar.gz" %}
-{%set hash_type = "sha256" %}
-{% set hash_val = "b6346459a15f0cbab01a45a90c3d25caf980d4733d628b4cc1952b865125d053" %}
-{%set build_num = "0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.{{ compress_type }}
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
-  {{ hash_type }}: {{ hash_val }}
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: "b6346459a15f0cbab01a45a90c3d25caf980d4733d628b4cc1952b865125d053"
 
 build:
-  number: {{ build_num }}
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  number: 0
```

```suggestion
  number: 1
```
As you heavily rewrite the recipe it's better to bump the build number. Then all python variants have the same recipe